docs/source/tutorials/resnet/index.md

```diff
@@ -12,48 +12,170 @@ LightlyTrain can be installed directly from PyPI:
 pip install lightly-train
 ```
 
-## Dataset Preparation
+## Pretrain ResNet with LightlyTrain
+
+### Download the Pretraining Dataset
+
+Here we use [a dataset for human detection on Kaggle](https://www.kaggle.com/datasets/killa92/human-detection-dataset) for pretraining. It contains 170 PNG images (of which 146 for training and 24 for validation) of various sizes containing humans.
```

Comment:
I would drop the information about train and validation images. It is not relevant for pretraining. It will only confuse users.